people/affiliations.json

```
@@ -182,5 +182,9 @@
     "alacuku": {
         "name": "Aldo Lacuku",
         "company": "Sysdig"
+    },
+    "rohith": {
```

```suggestion
    "rohith-raju": {
```
This needs to be your GH handle!